Configuration/Eras/python/Modifier_pps_2016_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
+
+pps_2016 =  cms.Modifier()
```

Comment:
there are also ```ctpps_201[678]``` modifiers.
Please do not replicate for increased confusion.

Reply:
@slava77 , @mundim , my impression that "Modifier_pps" is a more correct name than "Modifier_ctpps", because ctpps does not exist as a sub-detector.

Reply:
Modifier_pps_201X was a solution implemented back in 2018 to avoid conflict with the reconstruction, but I just tested it and apparently there is no problem. I would vote to keep the name as "Modifier_ctpps_201x" since it is already used by the reconstruction. It is planned an update in the detector name but I would prefer to postpone this change for when both simulation and reconstruction will implement the new name (pps) in a coordinate manner.

@jan-kaspar

Reply:
I also it think it makes sense to use the existing eras for now, and then refactor the era name later as a separate development item.

Reply:
I'm removing the duplicate Eras file (and making the needed adaptation in the related files), then the package (Configuration/Eras) will stay the same as in the release. Can I keep it in the same git branch or it is needed to close this PR and issue a new one with an cleanner branch?

Reply:
Done.

Reply:
moving from ctpps to pps should also be ok.
It's in the plans anyways.
@civanch  There was no pps detector when ctpps was initially created; it's a new name after the decision to merge the two collaborations.

Reply:
@mundim you can overwrite your current branch with a squashed commit, thereby updating this PR automatically.